### ISSUE TITLE
Monitoring of multiple events in one interface

### ISSUE THREAD

**Issue by frelion** [aio-libs/aiomonitor#402]
### Describe the solution you'd like

In my program, I use asyncio in multiprocessing. I don't want to create a separate interface for each process. Can I monitor them all in a single interface?

### Describe alternatives you've considered

Maybe the web server can be separate from the aiomonitor class, and the web server starts separately. The aiomonitor instance just sends messages to it.

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

**Comment by achimnol** [comment]
Yes, I agree with your request. Let me think a little bit more, but I cannot guarantee when it is going to be implemented.
